### PR TITLE
Add a simple HTTP server

### PR DESCRIPTION
**Pull request by brendandburns**
@quinton-hoole @wojtek-t

### CODE REVIEW

**Comment by quinton-hoole** [comment]
Doesn't this belong in kubernetes/contrib ?

File: perfdash/perfdash.go [hunk 1]
```diff
@@ -147,3 +174,39 @@ func findMethod(method, item string, data []APICallLatency) float64 {
...
+		fmt.Printf("Failed to get data: %v", err)
+		os.Exit(1)
+	}
+	if !*www {
+		generateCSV(buildLatency, resources, methods, os.Stdout)
+		return
+	}
+
+	go func() {
+		time.Sleep(10 * time.Minute)
```
Comment (quinton-hoole) [inline perfdash/perfdash.go]: Guess what I'm going to say :-)

**Comment by quinton-hoole** [comment]
Minor nit, otherwise LGTM.

File: perfdash/perfdash.go (hunk 1 above)
Comment (brendandburns) [inline perfdash/perfdash.go]: done (I think)

**Comment by brendandburns** [comment]
I added a full-on ux and docker image.  ptal.

--brendan

**Comment by brendandburns** [comment]
Demo is here:

http://104.197.215.156/

**Comment by wojtek-t** [comment]
Awesome!

LGTM

**Pull request merged**